Sources/Observers/KeyboardObserver.swift

```diff
@@ -53,11 +53,11 @@ public final class KeyboardObserver: NSObject {
 
     // MARK: - Private Methods
 
-    @objc private func keyboardDidShow() {
+    @objc private func keyboardBecameVisible() {
```

Comment:
as an alternative, maybe `handleKeyboardDidShow` / `handleKeyboardDidHide`? 💅 

additionally, since we're touching these and they are notification handling selectors, might as well implement a more correct® signature, by adding a `(_ notification: NSNotification)` parameter 🤓 

finally, perhaps leave a comment so that future devs don't feel tempted to "tidy" this to be `keyboardDidShow` and `keyboardDidHide` again 😅